piglow/examples/strobe.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"time"
+
+	"github.com/goiot/devices/piglow"
+
+	"golang.org/x/exp/io/i2c"
+)
+
+func main() {
+	piglow, err := piglow.Open(
```

Comment:
What about making it a single line? Not going to look verbose.

p, err := piglow.Open(&i2c.Devfs{Dev: "/dev/i2c-1", Addr: 0x54})